pykoi/rlhf/supervised_finetuning.py

```diff
@@ -163,6 +182,65 @@ def prepare_sample_text(self, example):
             f"    Answer: {example[self._rlhf_config.answer_title]}"
         )
         return text
+    
+
+    def prepare_d2l_text(self, example):
+        """Prepare the text from a sample of the d2l dataset ."""
+        INTRO_BLURB = (
+            "Below is an instruction that describes a task. Write a response that appropriately completes the request."
+        )
```

Comment:
qq that does `INTRO_BLURB` needed for SFT.

Unless user always put this as a part of their system prompt, I am wondering if user forget to include this as a part of their system prompt. It might hurt the inference performance.

Reply:
This is Yunfei's prompt and I kept it in order to reproduce his result in pykoi. I agree with the case you mentioned.